src/layers/core/polygon-layer/polygon-layer.js

```diff
@@ -82,6 +87,12 @@ export default class PolygonLayer extends Layer {
       pickingColors: {size: 3, type: GL.UNSIGNED_BYTE, update: this.calculatePickingColors, noAlloc}
     });
     /* eslint-enable max-len */
+
+    if (this.props.fp64) {
```

Comment:
+1.  Unfortunately we can't remove it at the moment if switching back but that is a rather unusual use case. Maybe leave a comment to that effect.